apps/code-infra-dashboard/netlify.toml

```diff
@@ -9,8 +9,8 @@
   # Default build command.
   command = "pnpm build"
 
-  # Deciden when to build Netlify
-  ignore = "cd ../../ && git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF apps/code-infra-dashboard pnpm-lock.yaml"
+  # Decide when to build Netlify
+  ignore = "cd ../../ && git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF $(pnpm ls --filter code-infra-dashboard --parseable --only-projects) pnpm-lock.yaml"
```

Comment:
@oliviertassinari Was this not working?

In https://github.com/mui/mui-public/pull/310 I'm changing the base dir, so I'm also removing the `cd ../..`